spec/compiler/macro/macro_methods_spec.cr

```diff
@@ -2247,6 +2247,29 @@ module Crystal
       end
     end
 
+    describe "metaclass methods" do
+      node = Metaclass.new(Path.new("Int32"))
```

Comment:
Instantiating this node is cheap. I'd prefer to do that in every example to keep them self-contained.